Automattic/wp-calypso
# Stats: Limit new feedback UI to paid commercial sites

Related to https://github.com/Automattic/red-team/issues/164

## Proposed Changes

Limits presentation of the new feedback UI (panel and inline card) to paid commercial sites.

## Why are these changes being made?


We want to start with a targeted release initially and can expand it as needed in future projects.

Part of the User Feedback project:

pejTkB-1Ce-p2

## Testing Instructions



* Test on a site with a commercial Stats plan.
* Visit the Traffic page.
* Enable the feature flag: `?flags=stats/user-feedback`
* Refresh the page and confirm the floating panel is presented.
* Confirm inline card is present at bottom of page.
* Test on a site without a commercial Stats plan.
* Confirm the floating panel is never presented and the inline card is hidden.
* Test on a DotCom site with Business plan.
* Confirm the floating panel is never presented and the inline card is hidden.

## Pre-merge Checklist
